tests/unit/drivers/complex_vlan.py

```diff
@@ -43,6 +43,9 @@ def config_initialize():
                         '{"name":"1", "switch":"test"}, ' \
                         '{"name":"2", "switch":"test"}]')
             cfg.set('driver complex_vlan', 'trunk_ports', '[]')
+            cfg.add_section('database')
+            cfg.set('database', 'sqlite', 'sqlite:///:memory:')
+            cfg.set('database', 'postgres', 'postgresql://postgres:postgres@localhost/haas')
```

Comment:
I would definitely add something to the effect that this is a missing feature (patches welcome). I'm not sure I want to be specifically advising people to go and change the hardcoded value every time they want to run the tests; I'd like to just put this near the top of the TODO list and have it solved soon.